src/Sylius/Bundle/AttributeBundle/Resources/config/doctrine/model/AttributeValue.orm.xml

```diff
@@ -21,7 +21,6 @@
             <generator strategy="AUTO" />
         </id>
 
-        <field name="text" column="text_value" type="text" nullable="true" />
```

Comment:
this value it's now coming from [AttributeValueTranslation.orm.xml](https://github.com/xantrix/Sylius/blob/c86d336da6b269263f3793a779cdf36ae0375f5e/src/Sylius/Bundle/AttributeBundle/Resources/config/doctrine/model/AttributeValueTranslation.orm.xml#L9-L8)